components/collective-page/sections/Contribute.js

```diff
@@ -224,17 +225,19 @@ class SectionContribute extends React.PureComponent {
                           </Box>
                         </Flex>
                       </ContainerSectionContent>
-
                       <ContributeCardsContainer ref={ref}>
-                        <ContributeCardContainer>
-                          <ContributeCustom
-                            collective={collective}
-                            contributors={financialContributorsWithoutTier}
-                            stats={contributorsStats}
-                            hideContributors={hasNoContributor}
-                            disableCTA={!collective.isActive}
-                          />
-                        </ContributeCardContainer>
+                        {!collective.settings.disableCustomContributions && (
```

Comment:
`settings` is not marked as `isRequired` in the props, so let's be safe

```suggestion
                        {!collective.settings?.disableCustomContributions && (
```

Alternative:
```
      settings: PropTypes.object.isRequired,
```